recipe/meta.yaml

```diff
@@ -21,7 +21,8 @@ build:
 requirements:
   build:
     - python
-    - nodejs
```

Comment:
Thanks for the info! If you think it's appropriate for conda-forge to have a stricter from-source policy like debian, that's fine, too, though not what I would choose. It's mainly my perspective as a receiver of notebook bug reports that I would like to avoid the opportunity for conda-installed notebook being allowed to produce different javascript results from pip-installed notebook.